src/components/code-snippet/__tests__/code-snippet-test-cases.js

```diff
@@ -83,4 +83,23 @@ testCases.maxHeight = {
   }
 };
 
+testCases.noCss = {
+  component: CodeSnippet,
+  description: 'CodeSnippet with edit buttons without CSS.',
+  props: {
+    code: fullHtml,
```

Comment:
Not sure if it's necessary, but you could consider using different example code for this test case because right now if you open in JSFiddle/CodePen you just see a blank page because the map is not visible without the CSS to position/size it.

Reply:
That's a good point! I was lazy and copy and pasted! I'll update.

Reply:
![image](https://user-images.githubusercontent.com/2180540/111180902-d5da5e00-8583-11eb-8456-9b57151917a1.png)